### PR TITLE
DMVP-1361: Renovate bot module

### PR DESCRIPTION
**Pull request by aghamyan44**

### CODE REVIEW

File: modules/renovate-bot/provider.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+provider "helm" {
+  kubernetes {
+    kubernetes {
```
Comment (aramkarapetian) [inline modules/renovate-bot/provider.tf:3]: Is this correct?

File: modules/renovate-bot/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+### Automated dependency updates. Multi-platform and multi-language.
+
+## Module Usage for GitLab
+```
+module "renovate-bot" {
+    source = "./"
```
Comment (aramkarapetian) [inline modules/renovate-bot/README.md:6]: Please set terraform registry value

File: modules/renovate-bot/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    
+    platform = "gitlab"
+    endpoint = "https://gitlab.example.com/api/v4"
+    
+    token    = "**********"
+    schedule = "0 1 * * *"
+    autodiscover  = true
+    
+    cluster_host = ""
```
Comment (aramkarapetian) [inline modules/renovate-bot/README.md:15]: Is this the way we decided to use providers?

File: modules/renovate-bot/renovate.tf [hunk 4]
```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "renovate" {
+  name      = "renovate"
+  namespace = "renovate"
```
Comment (mrdntgrn) [inline modules/renovate-bot/renovate.tf:3]: can we have helm release name and namespace as config variable, to allow to override when needed

File: modules/renovate-bot/config.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+renovate:
+    config: |
+        {
+            "platform": "${platform}",
+            "endpoint": "${endpoint}",
+            "token": "${token}",
+            "autodiscover": "${autodiscover}",
+        }
```
Comment (aramkarapetian) [inline modules/renovate-bot/config.yaml:8]: New line is missing here.

**Review by aramkarapetian — APPROVED** [review]
LGTM

**Pull request merged**